src/services/leave-service.ts

```diff
@@ -15,19 +17,26 @@ export class LeaveService {
   private publicHolidays:Array<PublicHoliday> = [];
   
   constructor(private http:Http) {
+    
   }
   
   getAvailableLeaves(id:any):Observable<AvailableLeaveCount> {
     let url = this.leaveServiceUrl + "/available?employeeId=" + id;
-    return this.http.get(url).map(res => res.json());
+    console.log(url);
```

Comment:
some of the changes are due to mergin develop in harshit branch so update for more files if changes are needed.